worldwide/road-network.json

```diff
@@ -234,6 +234,145 @@
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-0.107, 51.501]
+      },
+      "properties": {
+        "place_name": "London, UK",
+        "zoom": 8,
+        "place_description": "Low zoom highway styling."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-0.225, 51.575]
+      },
+      "properties": {
+        "place_name": "Brent Cross, London, UK",
+        "zoom": 14,
+        "place_description": "Mid zoom styling for motorway/trunks.",
+        "tags": [
+          "road:motorway",
+          "road:motorway_link",
+          "road:trunk",
+          "road:trunk_link"
+        ]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.667, 45.53]
+      },
+      "properties": {
+        "place_name": "Rose Quarter, Portland",
+        "zoom": 17,
+        "place_description": "Overlapping roads and tracks."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-75.143, 39.957]
+      },
+      "properties": {
+        "place_name": "Delaware Expressway, Philadelphia",
+        "zoom": 16,
+        "place_description": "One-way arrows on both highways and smaller roads."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.547, 37.853]
+      },
+      "properties": {
+        "place_name": "Marin Headlands, San Francisco",
+        "zoom": 14,
+        "place_description": "Tracks and paths.",
+        "tags": ["road:track"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-77.681, 39.766]
+      },
+      "properties": {
+        "place_name": "East Coast, United States",
+        "zoom": 8,
+        "place_description": "Shields at midzoom."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-117.811, 33.64]
+      },
+      "properties": {
+        "place_name": "Turtle Rock, Irvine",
+        "zoom": 16,
+        "place_description": "Turning circles."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-74.067, 40.739]
+      },
+      "properties": {
+        "place_name": "Pulaski Skyway, Croxton",
+        "zoom": 18,
+        "tags": ["road:construction"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.404, 37.783]
+      },
+      "properties": {
+        "place_name": "SoMa, San Francisco",
+        "zoom": 15,
+        "tags": ["road:service"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-77.044, 38.902]
```

Comment:
This coordinate location is pointing to a building. Could we update it to know if it is the road above or to the right?
<img width="478" alt="Screen Shot 2019-12-13 at 2 21 33 PM" src="https://user-images.githubusercontent.com/14204192/70836060-e8c81680-1db3-11ea-8beb-51ecec2cfdb0.png">